examples/nnunet_example/client.py

```diff
@@ -0,0 +1,177 @@
+import argparse
+import os
+import warnings
+from logging import INFO
+from os.path import exists, join
+from pathlib import Path
+from typing import Union
+
+with warnings.catch_warnings():
+    # Need to import lightning utilities now in order to avoid deprecation
+    # warnings. Ignore flake8 warning saying that it is unused
+    # lightning utilities is imported by some of the dependencies
+    # so by importing it now and filtering the warnings
+    # https://github.com/Lightning-AI/utilities/issues/119
```

Comment:
haahha seems like this is also from the pkg_resources module of setuptools. I was looking and it seems there was fix merged in for this that looks like its available in the [latest release](https://github.com/Lightning-AI/utilities/releases/tag/v0.11.6):
https://github.com/Lightning-AI/utilities/pull/281
also referenced here:
https://github.com/Lightning-AI/utilities/pull/292

Could be worth checking out if updating to the release fixes the error you are getting here

Reply:
So we are already on the newest version for lightning utilities so we'll probably have to wait until the next release comes out. I just checked and we definitely still need it. However it seems like the updated dependencies somehow got rid of the warnings I was getting from importing flwr so I can take that out and only silence the lightning utilities warnings

Reply:
Okay, no worries!